packages/forms/src/model.ts

```diff
@@ -1610,6 +1610,24 @@ export const FormControl: ɵFormControlCtor =
       }
     });
 
+interface UntypedFormControlCtor {
+  new(): UntypedFormControl;
+
+  new(formState?: any, validatorOrOpts?: ValidatorFn|ValidatorFn[]|FormControlOptions|null,
+      asyncValidator?: AsyncValidatorFn|AsyncValidatorFn[]|null): UntypedFormControl;
+
+  prototype: FormControl;
+}
+
+/**
+ * UntypedFormControl is a non-strongly-typed @see FormControl.
+ * This is currently used to facilitate the typed forms migration, as strict types for forms have
+ * not yet been merged.
+ */
```

Comment:
Since this is a part of the public API, I'd propose adding a note that it should not be used directly. Something like this:
```suggestion
/**
 * UntypedFormControl is a non-strongly-typed version of the `FormControl`.
 *
 * Note: it's used for migration purposes only. Please avoid using it directly in your code and prefer the `FormControl` instead.
 */
```

Reply:
Fixed.